.vscode/svelte.code-snippets

```diff
@@ -0,0 +1,262 @@
+{
+	// Place your living-dictionaries workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and 
```

Comment:
@jacob-8 what is this for?

Reply:
Gives you snippets that used to be provided by an extension. The extension grew stale and was missing important things like const and key so I just added the snippets to the repo and now the extension is no longer needed. Plus I have my favorite Vitest snippets in there too. Please try them out and use them.